src/slam_toolbox_common.cpp

```diff
@@ -319,8 +319,15 @@ bool SlamToolbox::shouldStartWithPoseGraph(
         pose.y = read_pose[1];
         pose.theta = read_pose[2];
       }
-    } else {
+    } else if (this->get_parameter("map_start_at_dock", start_at_dock)) {
```

Comment:
You already set start at dock in this line. The second line here is redundant. Perhaps there's a better way to do this